pinot-integration-tests/src/test/java/org/apache/pinot/integration/tests/BaseClusterIntegrationTestSet.java

```diff
@@ -623,8 +623,10 @@ public void testReload(boolean includeOfflineTable)
         JsonNode testQueryResponse = postQuery(testQuery);
         // Should not throw exception during reload
         assertEquals(testQueryResponse.get("exceptions").size(), 0);
-        // Total docs should not change during reload
-        assertEquals(testQueryResponse.get("totalDocs").asLong(), numTotalDocs);
+        // Total docs should eventually match
+        if (testQueryResponse.get("totalDocs").asLong() != numTotalDocs) {
```

Comment:
Why does the change in this PR need this change?

Reply:
This is the unit test for testing default values. When reload request is received, it takes time for pinot servers to update the schema. 
And this `TEST_DEFAULT_COLUMNS_QUERY = "SELECT COUNT(*) FROM mytable WHERE NewAddedIntDimension < 0"` in Line 640 queries a non-existing column in the new schema. That's why before the new schema gets fully reloaded in pinot-server, it'd return the totalDocs being not the same as the actual value of numTotalDocs.